auditbeat/auditbeat.reference.yml

```diff
@@ -1333,7 +1333,8 @@ logging.files:
 # Defines if the HTTP endpoint is enabled.
 #http.enabled: false
 
-# The HTTP endpoint will bind to this hostname or IP address. It is recommended to use only localhost.
+# The HTTP endpoint will bind to this hostname, IP address, unix socket or named pipe. It is recommended
```

Comment:
This doesn't quite make sense any longer. There's no concept of "localhost" for unix sockets. What if it read something like, `The HTTP endpoint will bind to this hostname, IP address, unix socket or named pipe. When using IP addresses, it is recommended to only use localhost`?

[This change would need to go in a few places in this PR if accepted.]

Reply:
changed in all the yaml.